docs/api/package_manager/base.md

```diff
@@ -73,6 +73,16 @@ def check_dependencies(configuration: PackageConfiguration) -> List[Tuple[Packag
 
 Verify hashes for package dependecies againts the available hashes.
 
+<a id="aea.package_manager.base.BasePackageManager.get_package_dependencies"></a>
+
+#### get`_`package`_`dependencies
+
+```python
+def get_package_dependencies(package_id: PackageId) -> List[PackageId]
+```
+
+Get package dependencies by package_id.
+
```

Comment:
this and the rest of api doc changes resulting from previous PR https://github.com/valory-xyz/open-aea/pull/483